ray/notebooks/Ray_RAPIDS_HPO.ipynb

```diff
@@ -85,10 +85,9 @@
     "from sklearn.model_selection import train_test_split as sktrain_test_split\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.        # TODO: cudf read_csv may be ready now?
</pre><p>Things look good so can we remove this TODO?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Removed.